### PR TITLE
Update icon used for deleting run tags

### PR DESCRIPTION
**Pull request by smurching**
## What changes are proposed in this pull request?

Updates icon used for deleting run tags from an antd-style trashcan to a font-awesome trash can

Before:
image

After:
image

Also updates confirmation prompt wording for tag deletion.

## How is this patch tested?

Manual testing

### Is this a user-facing change?

- [ ] No. You can skip the rest of this section.
- [x] Yes. Give a description of this change to be included in the release notes for MLflow users.

(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)

### What component(s) does this PR affect?

- [ ] UI
- [ ] CLI
- [ ] API
- [ ] REST-API
- [ ] Examples
- [ ] Docs
- [ ] Tracking
- [ ] Projects
- [ ] Artifacts
- [ ] Models
- [ ] Scoring
- [ ] Serving
- [ ] R
- [ ] Java
- [ ] Python

### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [ ] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

**Comment by gioa** [comment]
@smurching As we talked offline, we can use **fa-edit** and **fa-trash-o** from Font awesome for the icons.

**Comment by harupy** [comment]
FYI https://fontawesome.com/icons/trash-alt

> This icon replaces Font Awesome 4's fa-trash-o

**Comment by harupy** [comment]
`fa-edit` and `fa-trash-alt`

Screen Shot 2019-11-05 at 20 52 44

**Comment by gioa** [comment]
Thanks for the screenshot @harupy! Seems the line style of the Trash icon in Font Awesome 5 requires a pro license (see https://fontawesome.com/icons/trash-alt?style=light). 

I roughly remember we use Font Awesome 4? If it is 4 we are good with fa-trash-o?

**Comment by harupy** [comment]
@gioa 

> I roughly remember we use Font Awesome 4? If it is 4 we are good with fa-trash-o?

No, we use Font Awesome 5.
https://github.com/mlflow/mlflow/blob/master/mlflow/server/js/public/fontawesome/css/fontawesome-all.min.css#L2

**Comment by gioa** [comment]
@harupy Thanks for looking into this. Can we use this trash icon? 
https://fontawesome.com/icons/trash-alt?style=regular

**Comment by harupy** [comment]
@gioa I think we can. Let me try.

**Comment by harupy** [comment]
@gioa `far fa-trash-alt` looks much nicer.

far-trash-alt

File: mlflow/server/js/src/components/tables/EditableFormTable.js [hunk 1]
```diff
@@ -117,7 +117,7 @@ export class EditableTable extends React.Component {
               onConfirm={() => this.delete(record.key)}
             >
               <a disabled={editingKey !== ''}>
-                <i className="fas fa-times"/>
+                <i className="fas fa-trash-alt"/>
```
Comment (harupy) [inline mlflow/server/js/src/components/tables/EditableFormTable.js]: `fas` -> `far`?

File: mlflow/server/js/src/components/tables/EditableFormTable.js [hunk 2]
```diff
@@ -112,12 +112,12 @@ export class EditableTable extends React.Component {
             </a>
             <Popconfirm
               title="Are you sure you want to delete this tag？"
-              okText="Yes"
-              cancelText="No"
+              okText="Confirm"
```
Comment (mparkhe) [inline mlflow/server/js/src/components/tables/EditableFormTable.js]: Thank you!

**Comment by harupy** [comment]
@gioa @smurching Why downgrade Font Awesome from 5 to 4?

**Comment by smurching** [comment]
@harupy oops thanks for the catch - had accidentally included some extraneous fontawesome 4.7 files in the PR, deleted them :)

**Comment by harupy** [comment]
@smurching @gioa 
We will leave the edit icon as it is?

**Comment by smurching** [comment]
@harupy IMO the edit icon looks fine (although @gioa if you feel otherwise please lmk & I can make a follow-up!)

**Comment by gioa** [comment]
This edit icon from AntD looks fine with Font awesome icon. In the long term, it's better to use the same icon font library for consistency.

**Pull request merged**